build.sbt

```diff
@@ -6,6 +6,7 @@ name := "Agora"
 organization := "org.broadinstitute"
 
 scalaVersion := "2.11.12"
+dependencyOverrides += "org.scala-lang" % "scala-compiler" % "2.11.12"
```

Comment:
What compiler does SBT use to compile the code that selects the compiler? I guess SBT ships with a default/build-in one?

Reply:
I believe sbt-launcher has all the magic: http://www.scala-sbt.org/1.x/docs/Launcher-Getting-Started.html